components/script/dom/globalscope.rs

```diff
@@ -285,6 +285,14 @@ struct FileListener {
     task_canceller: TaskCanceller,
 }
 
+#[derive(JSTraceable, MallocSizeOf)]
```

Comment:
Since this struct will be used in the IPC router only, you don't need to derive those traits which are mostly meant for stuff inside a struct marked as `dom_struct`.